types/views/views.go

```diff
@@ -332,6 +333,26 @@ func (m Map[K, V]) GetOk(k K) (V, bool) {
 	return v, ok
 }
 
+// MarshalJSON implements json.Marshaler.
+func (m Map[K, V]) MarshalJSON() ([]byte, error) {
+	return json.Marshal(m.ж)
+}
+
+// UnmarshalJSON implements json.Unmarshaler.
+func (m *Map[K, V]) UnmarshalJSON(b []byte) error {
```

Comment:
The whole point of a view is to provide immutable access to the underlying type. Wouldn't `UnmarshalJSON` violate this goal?

Reply:
See Maisem's latest push.

Reply:
Fixed with an additional check. This is how we handle it in other places.
```
	if v.ж != nil {
		return errors.New("already initialized")
	}
```

Reply:
Should this use the `unmarshalJSON` helper method like most other code here?

Reply:
>Should this use the unmarshalJSON helper method like most other code here?

that only works for slices, I didn't spend too much time make it work on both slices and maps